CMakeLists.txt

```diff
@@ -1,46 +1,48 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 3.0)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
 
-# Set up project properties
-SET(PROJECT_NAMESPACE gepetto)
+# Project properties
+SET(PROJECT_ORG gepetto)
 SET(PROJECT_NAME example-robot-data)
 SET(PROJECT_DESCRIPTION "Set of robot URDFs for benchmarking and developed examples.")
-SET(PROJECT_URL https://gepgitlab.laas.fr/${PROJECT_NAMESPACE}/${PROJECT_NAME})
+SET(PROJECT_URL https://gepgitlab.laas.fr/${PROJECT_ORG}/${PROJECT_NAME})
 
+# Project options
+OPTION(BUILD_PYTHON_INTERFACE "Build the python unit tests and helpers" ON)
+OPTION(INSTALL_PYTHON_INTERFACE_ONLY "Install *ONLY* the python interface" OFF)
+
+# Project configuration
+IF(NOT INSTALL_PYTHON_INTERFACE_ONLY)
+  SET(PROJECT_USE_CMAKE_EXPORT TRUE)
+ENDIF(NOT INSTALL_PYTHON_INTERFACE_ONLY)
+SET(CUSTOM_HEADER_DIR ${PROJECT_NAME})
+
+# JRL-cmakemodule setup
 INCLUDE(cmake/base.cmake)
-INCLUDE(cmake/test.cmake)
 INCLUDE(cmake/python.cmake)
 
 # Print initial message
 MESSAGE("${PROJECT_DESCRIPTION}, version ${PROJECT_VERSION}")
```

Comment:
Can we turn these to `message(STATUS "")` please?

Reply:
Sure :)

Reply:
Thank you :-)